### PR TITLE
ADD: URv2 support

### PR DESCRIPTION
**Pull request by Overtorment**
TODO

* [x] read `UR:CRYPTO-ACCOUNT`
* [x] read PSBT
* [x] read `UR:CRYPTO-PSBT`
* [x]  display PSBT
* [x] display `UR:CRYPTO-ACCOUNT` as a cosigner
* [x] display ??? as a coordination setup
* [x] switch-toggle v1 vs v2 ..?
* [x] sort out dependencies

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xhjj09h8he2eejr3w50y0aya64?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/ur-v2-support-1622652892.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xfbydqfrhnfme4vkr3wt8ad9rr?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/ur-v2-support-1623254811.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1ccdbe51w7apen2rwfc0dkcp78?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/ur-v2-support-1623773907.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tvu9byaqp1a6y5gant2rwh2dgr?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/ur-v2-support-1623841932.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tn53158xdt4kb3t10fedpjdcqg?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/ur-v2-support-1624035684.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m28whte75wu1rzv6cdru83038r?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7154)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/152j4wbjhpbm4afqekx2jbbybc?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7155)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/j1dcuebtwpd67u4krprywa3ze0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7160)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8xk0t8p5rwp1ak6k8fbxyjw83g?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7187)

**Comment by Overtorment** [comment]
ready to merge. looks like 100% keystone compatibility. waiting for foundationdevices to report back

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bfxwmad0w1g02c4kjtwzpxgj2c?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7237)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zxw3dcb6mxujefc0a1pvajnthm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7260)

File: screen/settings/GeneralSettings.js [hunk 1]
```diff
@@ -85,6 +92,12 @@ const GeneralSettings = () => {
         <BlueText>{loc.settings.general_adv_mode_e}</BlueText>
       </BlueCard>
       <BlueSpacing20 />
+      <BlueListItem
+        Component={TouchableWithoutFeedback}
+        title="Legacy URv1 QR"
```
Comment (Overtorment) [inline screen/settings/GeneralSettings.js:97]: feels like can be left untranslated

Comment (Overtorment) [inline screen/settings/GeneralSettings.js:97]: feels like can be left untranslated

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/31k1azz7upbnpgdxefg9k3mmum?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7267)

**Review by marcosrdz — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Pull request merged**